vue-components/src/components/HelloWorld.vue

```diff
@@ -38,17 +38,30 @@ export default Vue.extend( {
 
 <style lang="scss">
 .wikit-HelloWorld {
+	inset: logical 0 5px 10px;
+
 	&__headline {
-		margin: 40px 0 0;
+		margin-block-start: 40px;
+		margin-inline-end: 0;
+		margin-block-end: 0;
```

Comment:
**Not really relevant - just for alignment**
```suggestion
		margin-block-start: 40px;
```

As soon we as we have #117 in place this will look a lot less convoluted as the opinionated reset will automatically set the 0 values for us.